### ISSUE TITLE
`make` `build-if-not-exists` and `build-if-changed` always build

### ISSUE THREAD

**Issue by mokagio** [Automattic/simplenote-electron#3211]
That's because they look for `dist/app.js` but webpack uses `fullhash` in the name, i.e. `dist/app.20fd562f4b5b27e29042.js`

https://github.com/Automattic/simplenote-electron/blob/f5d463f7a5f5beb3638696c9700b1c7781448744/Makefile#L21

https://github.com/Automattic/simplenote-electron/blob/f5d463f7a5f5beb3638696c9700b1c7781448744/webpack.config.js#L23

**Comment by mokagio** [comment]
> That's because they look for `dist/app.js` but webpack uses `fullhash` in the name

This behavior was introduced in https://github.com/Automattic/simplenote-electron/pull/1543

**Comment by codebykat** [comment]
Right, I don't think we want to revert the hash, rather improve the Makefile detection... I tinkered with it a bit in #3225 but I'm not sure if that helps or not.
